### PR TITLE
initialize state properly, when initialData is falsy

### PR DESCRIPTION
**Pull request by cherniavskii**

### CODE REVIEW

File: src/tests/useQuery.test.js [hunk 1]
```diff
@@ -506,4 +506,24 @@ describe('useQuery', () => {
     rendered.getByText('data')
     expect(rendered.getByTestId('isFetching').textContent).toBe('false')
   })
+
+  it('should initialize state properly, when initialData is falsy', async () => {
```
Comment (tannerlinsley) [inline src/tests/useQuery.test.js]:
```suggestion
  it('should initialize state properly, when initialData is undefined', async () => {
```

**Comment by tannerlinsley** [comment]
LMK when Ready

File: src/tests/useQuery.test.js (hunk 1 above)
Comment (cherniavskii) [reply src/tests/useQuery.test.js]: this test checks falsy, but defined initialData

Comment (tannerlinsley) [reply src/tests/useQuery.test.js]: Got it

File: src/tests/useQuery.test.js [hunk 2]
```diff
@@ -506,4 +506,24 @@ describe('useQuery', () => {
     rendered.getByText('data')
     expect(rendered.getByTestId('isFetching').textContent).toBe('false')
   })
+
+  it('should initialize state properly, when initialData is undefined', async () => {
```
Comment (cherniavskii) [inline src/tests/useQuery.test.js]:
```suggestion
  it('should initialize state properly, when initialData is falsy', async () => {
```

**Comment by cherniavskii** [comment]
Are we good?

**Comment by cherniavskii** [comment]
Looks like we're not :/

**Comment by cherniavskii** [comment]
I don't think we want to run tests before building

**Comment by tannerlinsley** [comment]
Yeah, do you want to update the changelog with a new version?

**Comment by cherniavskii** [comment]
Sure

**Pull request merged**